src/app/pages/download/DownloadAction.php

```diff
@@ -57,6 +58,23 @@ public function __invoke(Request $request, $response, $args)
     ]);
   }
 
+  private function getBanner(string $version, ReleaseType $rt): string
+  {
+    echo "<script>console.log('Debug Objects: " . $version . "' );</script>";
+    if (empty($version) || is_null($rt)) {
+      return '';
+    }
+    if ($rt->name() != ReleaseType::LTS()->name()) {
+      return $rt->banner();
+    }
+    foreach (ReleaseInfoRepository::getLongTermSupportVersions() as $ltsVersion) {
+      if ($ltsVersion->getVersion()->getMinorVersion() == $version) {
+        return $rt->banner();
+      }
+    }
+    return '<i class="si si-bell" style="background-color:#e62a10;"></i> <b style="color:#e62a10;">This LTS version is no longer supported.</b>';
```

Comment:
is the banner styling in $rt->banner() also made in php?

Reply:
[Yes](https://github.com/axonivy/website-developer/blob/fb5fd1ca4eeb70596a1015943548eaa5f6993ea1/src/app/domain/ReleaseType.php#L91)